tools/mo/openvino/tools/mo/ops/LSTM.py

```diff
@@ -68,3 +72,29 @@ def infer(node: Node):
         assert len(node.out_nodes()) <= 3
 
         rnn_infer(node, [1, 2])
+
+    @staticmethod
+    def reverse_infer(node: Node):
+        W_size = np.prod(node.in_port(1).data.get_shape())
+
+        multiplier = node.multiplier
+        hidden_size = node.hidden_size
+        num_layers = node.num_layers
+        direction = 2 if node.has_num_directions else 1
+
+        size = hidden_size * direction * multiplier
+        other_layer_params_size = (hidden_size * direction + hidden_size + 2) * size
+
+        first_layer_params_size = W_size - (num_layers - 1) * other_layer_params_size
+        # first_layer_params_size = (input_size + rnn_layer.hidden_size + 2) * size
```

Comment:
This line looks confusing. If it is necessary comment then, please, update them to make more clear. If not - remove it.

Reply:
agree, will update the comment

Reply:
done